docs/user/get-started.md

```diff
@@ -0,0 +1,333 @@
+
+        ```go
+        ids, train, test, err := load(datasetPath)
+        if err != nil {
+            glg.Fatal(err)
+        }
+        ```
+        </details>
+    3. Create the gRPC connection and Vald client with gRPC connection.
+        <details><summary>example code</summary><br>
+
+        ```go
+        ctx := context.Background()
+
+        conn, err := grpc.DialContext(ctx, grpcServerAddr, grpc.WithInsecure())
+        if err != nil {
+            glg.Fatal(err)
+        }
+
+        client := vald.NewValdClient(conn)
+        ```
+        </details>
+    4. Insert and Index
+    - Insert and Indexing 400 training datasets to the Vald agent.
+        <details><summary>example code</summary><br>
+
+        ```go
+        for i := range ids [:insertCount] {
+            if i%10 == 0 {
+                glg.Infof("Inserted %d", i)
+            }
+            _, err := client.Insert(ctx, &payload.Object_Vector{
+                Id: ids[i],
+                Vector: train[i],
+            })
+            if err != nil {
+                glg.Fatal(err)
+            }
+        }
+        ```
+        </details>
+    - Wait until indexing finish.
+        <details><summary>example code</summary><br>
+
+        ```go
+        glg.Info("Wait for indexing to finish")
+        time.Sleep(time.Duration(indexingWaitSeconds) * time.Second)
+        ```
+        </details>
+    5. search
+    - Seach 10 neighbor vectors fot each 20 test datasets and return list of neighbor vector.
+    - When getting approximate vectors, the Vald client sends search config and vector to the server via gRPC.
+        <details><summary>example code</summary><br>
+
+        ```go
+        glg.Infof("Start search %d times", testCount)
+        for i, vec := range test[:testCount] {
+            res, err := client.Seach(ctx, &payload.Search_Request){
+                Vector: vec,
+                Config: &payload.Search_Config{
+                    Num: 10,
+                    Radius: -1,
+                    Epsilon: 0.01,
+                }
+            }
+            if err != nil {
+                glg.Fatal(err)
+            }
+
+            b, _ := json.MarshalIndent(res.GetResults(), "", " ")
+            glg.Infof("%d - Results : %s\n\n", i+1, string(b))
+            time.Sleep(1 * time.Second)
+        }
+        ```
+        </details>
+
+    ```bash
+    # run example
+    go run main.go
+    ```
+
+## Advanced
+
+### Customize Vald
+
+Vald is highly customizable.
+For example you can configure the number of vector dimension, the number of replica and etc.
+You can customize Vald by creating/editing `values.yaml`.
+We will publish the instructions of `values.yaml` soon.
+
+### Another way to deploy Vald
+
+In the `Get Started` section, we'll show you how to deploy Vald with Scylla DB.
+Vald can be deployed in another way, which is used by Helm or `kind` command. (Compressor datastore is required, for example mysql + redis or casandora ).
```

Comment:
```suggestion
Vald can also be deployed by Helm or `kind` command. (Compressor datastore is required, for example mysql + redis or casandora ).
```

Reply:
can we add something like "for more information, please refer to xxx"?

Reply:
It's future work.
When we created the article about this, we'd like to update this sentence.